firmware/controllers/actuators/idle_thread.cpp

```diff
@@ -228,10 +228,19 @@ IIdleController::Phase IdleController::determinePhase(int rpm, int targetRpm, Se
 		return Phase::Coasting;
 	}
 
+	// If still in the cranking taper, disable closed loop idle
+	if (crankingTaperFraction < 1) {
+		return Phase::Running;
```

Comment:
Maybe we need a new separate phase for cranking-to-running transition state? Or just return it as Phase::Cranking here?

Reply:
yeah, I guess a separate phase is cleaner